<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QDialog" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Pomodoro_backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">
image:url(:/icon/icon/home.png);
background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="previousButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="stopButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="playButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="nextButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>176</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>174</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton1">
      <property name="styleSheet">
       <string notr="true">color:white;
background:#367a5f; border:0; border-radius:10; font: 20 bold &quot;Sherif&quot;; padding:5;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton3">
      <property name="styleSheet">
       <string notr="true">color:white;
background:#367a5f; border:0; border-radius:10; font: 20 bold &quot;Sherif&quot;; padding:5;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton2">
      <property name="styleSheet">
       <string notr="true">color:white;
background:#367a5f; border:0; border-radius:10; font: 20 bold &quot;Sherif&quot;; padding:5;</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>52</y>
     <width>174</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center  ; </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton1</sender>
   <signal>clicked()</signal>
   <receiver>playButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
